src/Behavior/MillisecondTimestamp.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ipl\Orm\Behavior;
+
+use DateTime;
+use DateTimeZone;
+use ipl\Orm\Contract\PropertyBehavior;
+
+class MillisecondTimestamp extends PropertyBehavior
+{
+    public function fromDb($value, $key, $context)
+    {
+        if ($value === null) {
+            return null;
+        }
+
+        $datetime = DateTime::createFromFormat('U.u', sprintf('%F', $value / 1000.0));
+        $datetime->setTimezone(new DateTimeZone(date_default_timezone_get()));
+
+        return $datetime;
+    }
+
+    public function toDb($value, $key, $context)
+    {
+        if ($value === null || $value === '*') {
```

Comment:
I didn't meant to specifically handle the asterisk with my other comment.

This now leads to exceptions in the searchbar if the user just starts typing.
There should be a test for `*now*` for example. (The searchbar adds those `*` automatically, there's no way of preventing this for the user)

Reply:
Why passing something like `null`, `*` and the other stuff to the behavior anyway? Every behavior needs special handling for that. Does that also mean that the `BoolBehavior` is broken?

Reply:
Please remove once #88 is merged.